dist/mercury.js

```diff
@@ -2581,7 +2585,7 @@ var NewrepublicComExtractor = {
     selectors: [['meta[name="og:image"]', 'value']]
   },
   content: {
-    selectors: [['.article-cover', 'div.content-body'], ['.minute-image', '.minutes-primary div.content-body']],
+    selectors: [['div.article-body']],
```

Comment:
Same as above, `dist/generate-custom-parser.j` is showing the previous classes...